prince/cross_sections/photo_meson.py

```diff
@@ -5,8 +5,8 @@
 
 from prince._deprecated.util import bin_widths, get_AZN
 from prince.data import spec_data
-from prince.util import info, load_or_convert_array
-from prince_config import config
```

Comment:
I think it would make the code more readable, if we only ever import the config as `import prince_config as config` and not from `prince_config import ...`.
That way it makes clearer where these options come from.

The way it is handeled in this module (and some others) it might be more difficult to trac for someone new to the code, where these come from and where to change them. (They essentially look like global variables).